fuel-client/assets/schema.sdl

```diff
@@ -241,6 +237,10 @@ enum ReturnType {
 	REVERT
 }
 scalar Salt
+enum SchemaCoinStatus {
```

Comment:
This isn't a very user friendly name, what was wrong with CoinStatus before?

Reply:
Reverted to `CoinStatus`